extensions/oauth/src/main/java/org/mule/extension/oauth2/internal/DeferredExpressionResolver.java

```diff
@@ -45,4 +46,15 @@ public DeferredExpressionResolver(MuleContext muleContext) {
     }
   }
 
+  public <T> String getExpression(ParameterResolver<T> expr) {
```

Comment:
I don't think there's an issue for this. I need the string for an expression, but putting the attribute in the POJO as String will try to resolve it.